src/components/CurrentScenarioSupersetReport/CurrentScenarioSupersetReport.js

```diff
@@ -17,6 +17,7 @@ const CurrentScenarioSupersetReport = ({
   reportConfiguration,
   index,
   labels,
+  contextType,
```

Comment:
It seems that `contextType` is not used in this file, shouldn't it be forwarded to `useCurrentScenarioSupersetReport`?